src/client/debugger/debugAdapter/DebugClients/LocalDebugClient.ts

```diff
@@ -84,15 +74,16 @@ export class LocalDebugClient extends DebugClient<LaunchRequestArguments> {
                 pythonPath = this.args.pythonPath;
             }
             const args = this.buildLaunchArguments(processCwd, dbgServer.port);
+            const envVars = this.args.env ? { ...this.args.env! } : {};
```

Comment:
```suggestion
            const envVars = this.args.env ? { ...this.args.env } : {};
```

Since we've already checked whether `this.args.env` is not undefined, we won't need the `!`